src/worker.py

```
@@ -50,6 +50,17 @@
 from pyodide.ffi import to_js
 from js import WebSocketPair, WebSocketRequestResponsePair
 import uuid
+try:
+    from chat_do import ChatDO as _ChatDO
+except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)